<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatosPersonales</class>
 <widget class="QDialog" name="DatosPersonales">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnGuardar">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>220</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="icon">
    <iconset resource="imagenes/imagenes.qrc">
     <normaloff>:/save.jpeg</normaloff>:/save.jpeg</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Volver</string>
   </property>
   <property name="icon">
    <iconset resource="imagenes/imagenes.qrc">
     <normaloff>:/atras.png</normaloff>:/atras.png</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,1,5">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nombre      </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtNombre"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Apellidos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtApellidos"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Dirección    </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtDireccion"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2,1,4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>C.P.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtCodigoPostal">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Población</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtPoblacion"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Provincia    </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtProvincia"/>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>País</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtPais"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="imagenes/imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
